services/self-service/src/main/java/com/epam/dlab/backendapi/service/impl/ProjectServiceImpl.java

```diff
@@ -344,10 +344,14 @@ private String getUpdateBudgetAudit(ProjectDTO p) {
 		if (!configuration.isAuditEnabled()) {
 			return null;
 		}
-		Integer value = Optional.ofNullable(get(p.getName()).getBudget())
+		ProjectDTO projectDTO = get(p.getName());
+		Integer value = Optional.ofNullable(projectDTO.getBudget())
 				.map(BudgetDTO::getValue)
 				.orElse(null);
-		return String.format(AUDIT_UPDATE_BUDGET, value, p.getBudget().getValue());
+		boolean monthlyBudget = Optional.ofNullable(projectDTO.getBudget())
```

Comment:
Can you use primitive here? Because you assign **null** in **orElse(null)**

Reply:
Primitive has been changed